java/maven/src/org/netbeans/modules/maven/classpath/MavenSourcesImpl.java

```diff
@@ -78,7 +78,7 @@
 public class MavenSourcesImpl implements Sources, SourceGroupModifierImplementation, OtherSourcesExclude {
     public static final String TYPE_OTHER = "Resources"; //NOI18N
     public static final String TYPE_TEST_OTHER = "TestResources"; //NOI18N
-    public static final String TYPE_GEN_SOURCES = "GeneratedSources"; //NOI18N
```

Comment:
This may lead to a possible incompatibility, if someone looks for "GeneratedSources". Would it be possible to handle both type strings ?